testsuite/microprofile-tcks/fault-tolerance/pom.xml

```diff
@@ -99,6 +99,11 @@
                      <name>java.util.logging.config.file</name>
                      <value>${project.build.testOutputDirectory}/logging.properties</value>
                   </property>
+                  <!-- Uncomment to debug arquillian test ->
```

Comment:
Looks like this is what's causing the pom parsing to fail. Needs `-->` instead at the end